enhancements/console/dynamic-plugins.md

```diff
@@ -147,6 +147,44 @@ spec:
     basePath: '/'
 ```
 
+In case the plugin needs to communicate with an in-cluster services, it can
+declare the communication in it's `ConsolePlugin` manifest, defining
+`spec.proxyService` array field entry. In there service `name`,
+`namespace`, `port` and `basePath` needs to specified. The console backend
+will proxy the plugin request to the dependent service using the service
+CA bundle.
```

Comment:
Do we require https? It would be good to state that here if so.

Reply:
> Your example has a CA certificate. So I'm assuming we don't always use the service CA bundle?

So I thought we wanna use the custom service-ca if the service is using a custom CA. If not we will default to the  service CA bundle.

> Do we require https? It would be good to state that here if so.

So when we are creating a service endpoint thats constructed from <service-name>-<service-namespace>:<service-port> it needs to be https. So I guess the https is requred otherwise the the service server will respond with `http: server gave HTTP response to HTTPS client`